tmk_core/common/usb_util.c

```diff
@@ -13,8 +13,8 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+#include "quantum.h"
```

Comment:
Not a fan of this tbh.

Reply:
I would like it to be just gpio/pin defs, but the include order isnt consistent enough.

gpio needs another refactor pass anyway, so i hope to cover it then.